docs/worldgen/biomemodifier.md

```diff
@@ -0,0 +1,648 @@
+# Biome Modifiers
+
+Biome Modifiers are a data-driven system that allows for changing many aspects of a biome. Ranging from injecting or removing PlacedFeatures, adding or deleting mob spawns, changing the climate, and adjusting foliage and water color. NeoForge provides several default Biome Modifiers that covers the majority of use cases for both players and modders.
+
+For players and pack makers, you will want to see the [#Applying-Biome-Modifiers](#Applying-Biome-Modifiers) section and [#Builtin-Neoforge-Biome-Modifiers](#Builtin-Neoforge-Biome-Modifiers) section.
+
+For modders looking to do basic goals, you will want to see the [#Applying-Biome-Modifiers](#Applying-Biome-Modifiers) section, [#Builtin-Neoforge-Biome-Modifiers](#Builtin-Neoforge-Biome-Modifiers) section, and possibly the [#Datagenning-Biome-Modifiers](#Datagenning-Biome-Modifiers) section if you prefer to datagen.
+
+For modders looking to do advance custom Biome Modifiers, check out the [#Applying-Biome-Modifiers](#Applying-Biome-Modifiers) section and [#Creating-Custom-Biome-Modifiers](#Creating-Custom-Biome-Modifiers) section.
+
+## Applying Biome Modifiers:
+
+To apply a Biome Modifier, the JSON file will need to be under `data/<modid>/neoforge/biome_modifier/<path>.json` folder in the mod's resources or in a datapack. Neoforge will automatically detect and load the Biome Modifier and apply it to the game. Existing Biome Modifiers can be overridden by having a new JSON file at the exact same location and name.
```

Comment:
And for what a Biome Modifier is, that's explains in the first original short summary at top. This is more for how to make the file be loaded. Which for players/packmakers, they need to know the location filepath. I can try rewording a little but this isn't supposed to explain biome modifiers as that's covered elsewhere

Reply:
The way this is worded just reminds me of questionable youtube tutorials ~~mine included~~ that just tell you to copy paste. I honestly think a bit of fluff would just help it transition better instead of immediately starting out with the theme.